test/client/fixtures/sandbox/code-instrumentation/location-test.js

```diff
@@ -157,6 +178,54 @@ test('special pages (GH-339)', function () {
     destLocation.forceLocation(storedForcedLocation);
 });
 
+test('should add the trailing slash to location.href if url consist of origin (GH-1426)', function () {
+    var storedForcedLocation = destLocation.getLocation();
+
+    function checkTrailingSlash (urls, trailingSlashIsNeeded) {
+        urls.forEach(function (url) {
+            destLocation.forceLocation(urlUtils.getProxyUrl(url));
+
+            var windowMock = {
+                location: {},
+                document: document
+            };
+
+            var locationWrapper = new LocationWrapper(windowMock);
+
+            strictEqual(locationWrapper.href, url + (trailingSlashIsNeeded ? '/' : ''));
```

Comment:
It's a wrong test because we add a trailing slash on `href` property setter.

Reply:
Done, new test: 'should add the trailing slash to location.href using href.set(), assign() and replace() functions if url consist of origin (GH-1426)'